.github/workflows/ci.yml

```diff
@@ -66,7 +66,7 @@ jobs:
       run: |
         echo "Checking that code generation is up-to-date"
         go install github.com/golang/mock/mockgen@v1.6.0
-        go install github.com/deepmap/oapi-codegen/cmd/oapi-codegen@v1.10.1
+        go install github.com/andrewstucki/oapi-codegen/cmd/oapi-codegen@indirection
```

Comment:
Just FYI, this is a temporary fork while this PR winds up getting worked out upstream: https://github.com/deepmap/oapi-codegen/pull/382 -- basically it makes it so we don't have an unneeded indirection for the `object`/`map` fields like `Meta` in the generated structs.